WordPress/Jetpack/Resources/AppStoreStrings.po

```diff
@@ -0,0 +1,109 @@
+msgstr ""
+
+msgctxt "v17.6-whats-new"
+msgid ""
+"With 17.6, you can add the Audio Block to any WordPress.com site, whatever plan you’re on – including the Free Plan. The Audio Block makes it easy to add podcasts, music, and sound files to your posts and pages.\n"
+"\n"
+"We’ve been hard at work wrangling UX improvements, too:\n"
+"\n"
+"When you assign a parent page to a page without a title, you’ll now see a ‘(no title)’ placeholder instead of a blank line, giving you a gentle nudge to name the page.\n"
+"\n"
+"You’ve never been able to edit Post Archive pages because they’re automatically generated. In 17.6, you won’t be able to open them in the editor, either, saving you a wasted trip.\n"
+"\n"
+"We’ve also been working on fixing things:\n"
+"\n"
+"We took care of some notices that had been overlapping the ActionSheet, including one that displays Site Icon controls, another displaying the QuickStart Removal, and a third that displays More Actions in the editor.\n"
+"\n"
+"Last up, we squashed a bug that sometimes caused the app to crash during comment moderation and another to fix a login error for WordPress.org sites to show inline.\n"
+"\n"
+"That’s all for 17.6. See you in 17.7!\n"
+msgstr ""
+
+#. translators: This is a promo message that will be attached on top of the first screenshot in the App Store.
+#. No specified characters limit here, but try to keep as short as the source one.
+msgctxt "screenshot-text-1"
+msgid ""
+"Safer.\n"
+"Faster.\n"
+"More traffic.\n"
+msgstr ""
+
+#. translators: This is a promo message that will be attached on top of the second screenshot in the App Store.
+#. No specified characters limit here, but try to keep as short as the source one.
+msgctxt "screenshot-text-2"
+msgid ""
+"Bring your\n"
+"Jetpack with you.\n"
+msgstr ""
+
+#. translators: This is a promo message that will be attached on top of the third screenshot in the App Store.
+#. No specified characters limit here, but try to keep as short as the source one.
+msgctxt "screenshot-text-3"
+msgid ""
+"Keep tabs on\n"
+"your site activity.\n"
+msgstr ""
+
+#. translators: This is a promo message that will be attached on top of the fourth screenshot in the App Store.
+#. No specified characters limit here, but try to keep as short as the source one.
+msgctxt "screenshot-text-4"
+msgid ""
+"Scan for issues\n"
+"on the go.\n"
+msgstr ""
+
+#. translators: This is a promo message that will be attached on top of the fifth screenshot in the App Store.
+#. No specified characters limit here, but try to keep as short as the source one.
+msgctxt "screenshot-text-5"
+msgid ""
+"Back Up your site\n"
+"at any moment.\n"
+msgstr ""
+
+#. translators: This is a promo message that will be attached on top of the fifth screenshot in the App Store.
+#. No specified characters limit here, but try to keep as short as the source one.
+msgctxt "screenshot-text-6"
+msgid ""
+"Site stats in your\n"
+"pocket.\n"
+msgstr ""
+
```

Comment:
The release toolkit adds this new line, which really bothers me... I'm not sure why that happens here and not in the other file. 

Any ideas?